Dockerfile

```diff
@@ -1,9 +1,18 @@
-FROM python:3.8.13
+FROM python:3.8.13-slim-bullseye
 
-ENV PIP_NO_CACHE_DIR off
-ENV PIP_DISABLE_PIP_VERSION_CHECK on
-ENV PYTHONUNBUFFERED 1
-ENV PATH="/usr/src/app/bin:${PATH}:/opt/google-cloud-sdk/bin"
+ENV PIP_NO_CACHE_DIR=off \
+    PIP_DISABLE_PIP_VERSION_CHECK=on \
+    PYTHONUNBUFFERED=1 \
+    VOLTA_HOME=/.volta \
+    VOLTA_VERSION=0.8.1 \
+    VOLTA_SHA256=e1a45f073580552318c206069e5bf0dd8cb9dcdc1bfeaecc5a96c48a5208dd7a \
+    SENTRY_CLI_VERSION=1.69.1 \
+    SENTRY_CLI_SHA256=4bed363e76e853aa1855b228b73b1e13a6b71209ce699bb0a117f98d6cfd8962 \
+    DOCKER_CLI_VERSION=20.10.16 \
+    DOCKER_CLI_SHA256=96588db31509c2a3c89eb68107b9bb9a0e6b1c9b5791e5c18475680d5074b40f \
+    GCLOUD_VERSION=382.0.0 \
+    GCLOUD_SHA256=335e5a2b4099505372914acfccbb978cf9d4efd8047bda59f910c26daefd554e \
```

Comment:
these are better set as `ARG`s so they don't pollute the container environment

Reply:
Oh, nice tip. I'm gonna f/u everywhere to do that + move stuff over to dev-infra-assets bucket.